pages/firststeps.md

```diff
@@ -61,9 +61,9 @@ Follow the directions at [Git Repositories](gitandrepositories.md) to keep your
 
 Complete the tutorial under the [GitHub Issues tab](githubissues.md) and make sure to keep us posted in the Gitter chat. You should link in Gitter whenever you create an issue or when you comment on someone else's issue. Post as many issues as you can for improving the page as well as personal practice. If you know how to solve an issue, be sure to provide a detailed account of your research and show how to fix it. Next, fix the issue on your repo and create pull requests. Don't forget you can also comment and work on issues that you didn't create. The more you work and commit and push, the more impressive your GitHub profile will become. This is very important to potential employers, thus we encourage you to spend time to increase your numbers to strengthen your profile.
 
-## Step 7 - Nation BeLL:
+## Step 7 - Nation BeLLs:
```

Comment:
@praharpatel please remove this file - cause it has nothing to do with your personal page

other question @praharpatel are you familiar with git branches?